glass/src/libnt/native/cpp/NetworkTables.cpp

```diff
@@ -808,13 +809,15 @@ void NetworkTablesModel::Update() {
             m_server.publishers.clear();
           } else if (info->name == "$serversub") {
             m_server.subscribers.clear();
-          } else if (wpi::starts_with(info->name, "$clientpub$")) {
-            auto it = m_clients.find(wpi::drop_front(info->name, 11));
+          } else if (auto client =
+                         wpi::remove_prefix(info->name, "$clientpub$")) {
+            auto it = m_clients.find(client);
```

Comment:
```suggestion
            auto it = m_clients.find(*client);
```
Not sure why this wasn't a compiler error?

Reply:
Looks like `optional` has [comparison operators](https://en.cppreference.com/w/cpp/utility/optional/operator_cmp) against the wrapped type and [`std::map.find()`](https://en.cppreference.com/w/cpp/container/map/find) can accept types that can be compared to the key type, so maybe that's what was going on? (I'm still not sure about the `Compare::is_transparent` stuff, but it seems possible and we don't need to worry about it anyways)